iGCS/Utils/SettingsData.m

```diff
@@ -0,0 +1,89 @@
+//
+//  SettingsData.m
+//  iGCS
+//
+//  Created by David Leskowicz on 2/14/15.
+//
+//
+
+#import "SettingsData.h"
+#import "FileUtils.h"
+
+@implementation SettingsData
```

Comment:
Change name to GCSSettings and move the file to the model folder in the project. If you won't see that folder you should probably merge master into your branch at this point.